scripts/validate_docstrings.py

```diff
@@ -258,6 +259,9 @@ def pandas_validate(func_name: str):
                     pandas_error("EX04", imported_library=wrong_import)
                 )
 
+    if "array_like" in doc.raw_doc:
+        result["errors"].append(pandas_error("GL05"))
```

Comment:
currently, no test covers this - could you add one in `tests/scripts/validate_docstrings.py`?